scripts/populate_mock_db.py

```
@@ -0,0 +1,111 @@
+
+    c.executemany(
+        "INSERT INTO crew (name, role, skills, contract_hours) VALUES (?, ?, ?, ?)",
+        crew_data,
+    )
+
+    # Add appliance
+    c.execute("INSERT INTO appliance (name) VALUES ('P22P6')")
+    appliance_id = c.lastrowid
+
+    # Add availability
+    now = datetime.now()
```

Ruff (0.15.1)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)